server/controllers/events/events_controller.go

```diff
@@ -116,6 +116,7 @@ func NewVCSEventsController(
 	scope tally.Scope,
 	githubWebhookSecret []byte,
 	allowDraftPRs bool,
+	prReviewCommandRunner handlers.PRReviewCommandRunner,
```

Comment:
thoughts on embedding the `PRReviewCommandRunner` into `events.CommandRunner` so that we don't have to pass an extra argument here? This should not break any existing code imo

Reply:
Good callout, merged them.